api/lib/use-cases/CreateSharedPlan.js

```diff
@@ -11,13 +11,20 @@ module.exports = ({ fetchRecords, sharedPlan }) => {
       Object.values(record.systemIds)
     );
 
+    const jigsawCaseId = record.housingNeeds
+      ? record.housingNeeds.jigsawCaseId
+        ? true
+        : false
+      : false;
```

Comment:
```suggestion
    const jigsawCaseId = record.housingNeeds 
      && record.housingNeeds.jigsawCaseId ? true : false;
```
Will avoid the double ternary expression :)